draft-birkholz-cose-cometre-ccf-profile.md

```diff
@@ -98,29 +111,42 @@ where:
 - : denotes concatenation of lists
 - D\[k1:k2\] = D'_(k2-k1) denotes the list \{d'\[0\] = d\[k1\], d'\[1\] = d\[k1+1\], ..., d'\[k2-k1-1\] = d\[k2-1\]\} of length (k2 - k1).
 
-## Leaf Components
+## Transaction Components
 
 Each leaf in a CCF ledger carries the following components:
 
 ~~~
 CCF-leaf = [
-  internal-hash: bstr ; a string of HASH_SIZE bytes;
-  internal-data: bstr; a string of at most 1024 bytes; and
-  data_hash: bstr ; the serialization of the element stored at this leaf.
+
+  ; a string of HASH_SIZE bytes
+  internal-transaction-hash: bstr
+
+  ; a string of at most 1024 bytes;
+  internal-evidence: bstr
+
+  ; a string of HASH_SIZE bytes
+  data-hash: bstr
 ]
 ~~~
 
-The `internal_hash` and `internal_data` byte strings are internal to the CCF implementation. Similarly, the auxiliary tree entries are internal to CCF. They are opaque to receipt Verifiers, but they commit the TS to the whole tree contents and may be used for additional, CCF-specific auditing.
+The `internal-transaction-hash` and `internal-evidence` byte strings are internal to the CCF implementation. They can be safely ignored by receipt Verifiers, but they commit the TS to the whole tree contents and may be used for additional, CCF-specific auditing.
 
 # CCF Inclusion Proofs
 
 CCF inclusion proofs consist of a list of digests tagged with a single left-or-right bit.
 
 ~~~
-CCF-inclusion-proof: [+ proof-element],
+CCF-inclusion-proof: [
+  leaf (label TBD_2): CCF-leaf
+  path (label TBD_3): [+ ccf-proof-element]
+]
+
+ccf-proof-element = [
 
-proof-element = [
+  ; position of the element
   left: bool
+
```

Comment:
```suggestion
```
Why newlines? For readability? Maybe appending the comments behind the array members combines readability and no newlines in the array definition.

Reply:
I adopted the style I found in https://datatracker.ietf.org/doc/draft-ietf-cose-hash-envelope/, but I am super happy to line up on what's generally preferred.

Reply:
Reverted to something more compact.